src/checkbox-sk/checkbox-sk_test.ts

```diff
@@ -22,17 +24,17 @@ afterEach(() => {
   container.innerHTML = '';
 });
 
-function _checkBoxSetup() {
+function _checkBoxSetup(): Promise<CheckOrRadio> {
   return window.customElements.whenDefined('checkbox-sk').then(() => {
```

Comment:
These could also be changed to async/await, not sure if that would be any more readable.

Reply:
100% agree that async/await is more readable.